computestorage/helpers.go

```diff
@@ -72,9 +72,7 @@ func SetupContainerBaseLayer(ctx context.Context, layerPath, baseVhdPath, diffVh
 		if err != nil {
 			syscall.CloseHandle(handle)
 			os.RemoveAll(baseVhdPath)
-			if os.Stat(diffVhdPath); err == nil {
-				os.RemoveAll(diffVhdPath)
-			}
+			os.RemoveAll(diffVhdPath)
```

Comment:
Instead of removing the `Stat` call, should we be checking the return value?

Reply:
I didn't really even see a need for the stat call. If the file doesn't exist RemoveAll will just silently fail anyways.